_posts/2015-05-15-download-setup.md

```diff
@@ -94,6 +94,11 @@ dependencies {
   kapt 'com.github.bumptech.glide:compiler:4.1.1'
 }
 ```
+Note that you must also include the ``kotlin-kapt`` plugin in your ``build.gradle`` file:
```

Comment:
double-tick is strange, but I guess it still renders correctly.

Reply:
@TWiStErRob they're used all over, I also thought they were strange, but didn't want to come in and start changing convention!

Reply:
Ah, k. 👍

Reply:
For my own curiosity, is there a better way of getting the same effect?

Reply:
Yes, single backticks :)

For some reason GitHub supports any number of ticks:
```
`code`
``code``
```code```
```````code```````
```
All render as `code`, ``code``, ```code```, ```````code``````` inline text. Notice that even 3 can be used as inline. However, I don't think this is a documented feature. Read "Styling with Markdown is supported" below the text area for adding comments. That and other docs all say use single for inline and GFM gives you 3 for code blocks.